AEPMessaging/Sources/Event+Messaging.swift

```diff
@@ -151,12 +151,8 @@ extension Event {
         return nil
     }
 
-    private var messageCornerRadius: CGFloat? {
-        if let radius = mobileParametersDictionary?[MessagingConstants.Event.Data.Key.IAM.CORNER_RADIUS] as? Double {
-            return CGFloat(radius)
-        }
-
-        return nil
+    private var messageCornerRadius: Int? {
```

Comment:
🐛  - wrong type for corner radius

Reply:
Shouldn't the corner radius be CGFloat?
https://developer.apple.com/documentation/quartzcore/calayer/1410818-cornerradius